aspnetcore/grpc/troubleshoot/sample/8.0/GrpcGreeterClient/Program.cs

```diff
@@ -0,0 +1,203 @@
+﻿#define StandardHTTPS  // Options: StandardHTTPS | IgnoreInvalidCertificate | IgnoreInvalidCertificateClientFactory | CallInsecureGrpcServices | DotNet3InsecureGrpcServices | SubdirectoryHandler
```

Comment:
Why do you need this? Get them all to compile.

Reply:
OK, thanks, I will see what is reasonable to bundle together in the single gRPC client app.

Reply:
Following your suggestion, I did try combining different groups of them, It sounds like a good rule of thumb, thanks, but in this case, I realized by doing so I was making what was really clear cut for each example, a little harder to follow for the customer by grouping them.  As they are you can tell exactly what each is requiring up front for using statements such as the possible group for ignoring invalid certificates.  For a possible group of troubleshooting an insecure service it messes with the example a bit to account for more than one channel or to remove that before starting another.  Same related issues for Subdirectory handler vs Http3Handler.  So, I'm making the call to not complicate it for little value and keep it clean and comparable in the doc using the same variable names, etc. in each.